src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-list/cephfs-list.component.ts

```diff
@@ -69,8 +85,32 @@ export class CephfsListComponent extends ListWithDetails implements OnInit {
         icon: Icons.add,
         click: () => this.router.navigate([this.urlBuilder.getCreate()]),
         canBePrimary: (selection: CdTableSelection) => !selection.hasSelection
+      },
+      {
+        name: this.actionLabels.EDIT,
+        permission: 'update',
+        icon: Icons.edit,
+        click: () => this.editAction()
+      },
+      {
+        permission: 'delete',
+        icon: Icons.destroy,
+        click: () => this.removeVolumeModal(),
+        name: this.actionLabels.REMOVE,
```

Comment:
Even though the button says `Remove`, the modal heading and button says `Delete Volume`. That should also be titled with `Remove Volume`

Reply:
Fixed!